### DISCUSSION TITLE
Multiple led indicators at the same time? (usb Host connected to HID keyboard) rp2040

### DISCUSSION THREAD

**Discussion by metalex201** [hathach/tinyusb#1573] [Q&A] [answered]
Hello i'm trying to change multiple led indicators using the host_cdc_hid example from the pico-examples (rp2040)

```
static uint8_t leds = KEYBOARD_LED_CAPSLOCK;
tuh_hid_set_report(dev_addr, instance, 0, HID_REPORT_TYPE_OUTPUT, &leds, sizeof(leds));
```
I know that i can turn on any led by just changing the **leds** variable and running **tuh_hid_set_report** again,
But i struggle to figure out any way i could have multiple indicator leds on at the same time. Is this possible in any way?
And when the **tuh_hid_set_report** is called does it run **tuh_hid_report_received_cb**? 
(cause i always end up causing a closed loop if i put **tuh_hid_set_report** in **process_kbd_report** )
(I'm a begginner at coding in C, so a simple example would be apreciated)
The last discussion regarding this topic is a year old ( and mostly answered ) so i hope i'm not doing this incorrectly by making a new one (apologies if it is)

**Comment by hathach** [comment, marked as answer]
which indicator you want to also want to set, just use or logic 

```C
static uint8_t leds = KEYBOARD_LED_CAPSLOCK | ANOTHER_INDICATOR
```

**Reply by metalex201** [reply]
Thanks for the help! It's now working.
